service/dap/types.go

```diff
@@ -117,19 +117,18 @@ type LaunchConfig struct {
 	// This is required for "core" mode but unused in other modes.
 	CoreFilePath string `json:"coreFilePath,omitempty"`
 
-	// DelveCwd is the working directory of Delve.
-	// If specified, the delve DAP server will change its working
+	// DlvCwd is the new working directory for dlv server.
+	// If specified, the server will change its working
 	// directory to the specified directory using os.Chdir.
-	// Any relative paths used in most of other attributes are
-	// interpreted as paths relative to Delve's working directory
-	// unless explicitely stated otherwise. When Delve needs to
-	// build the program (in debug/test modes), Delve runs the
-	// go command from the Delve's working directory.
+	// Any other launch attributes with relative paths interpreted
+	// using dlv's working directory will use this new directory.
+	// When dlv needs to build the program (in debug/test modes),
+	// it will run the go command from this directory as well.
 	//
-	// If a relative path is provided as DelveCwd, it will be
-	// interpreted as a path relative to Delve's current working
+	// If a relative path is provided as DlvCwd, it will be
+	// interpreted as a path relative to dlv's current working
 	// directory.
-	DelveCwd string `json:"delveCwd,omitempty"`
+	DlvCwd string `json:"dlvCwd,omitempty"`
```

Comment:
Another option is DelveCwd (DelveCWD?) as a Go struct field name, and `dlvCwd` as json field name. Up to you.